docs/src/content/docs/fr/showcase.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Vitrine Starlight
+description: Découvrez les sites construits avec Starlight et les outils communautaires qui étendent Starlight !
+---
+
+:::tip[Ajoutez le vôtre !]
+Avez-vous construit un site Starlight ou un outil pour Starlight ?
+Ouvrez une PR en ajoutant un lien à cette page !
+:::
+
+## Sites
+
+import Card from '../../../components/showcase-card.astro';
+import FluidGrid from '../../../components/fluid-grid.astro';
+
+Starlight est déjà utilisé en production. Voici quelques sites sur le web :
+
+<FluidGrid>
+	<Card
+		title="Athena OS"
+		href="https://www.athenaos.org"
+		thumbnail="www.athenaos.org.png"
+	/>
+	<Card
+		title="PubIndexAPI Docs"
+		href="https://docs.pubindexapi.com/"
+		thumbnail="docs.pubindexapi.com.png"
+	/>
+	<Card
+		title="pls"
+		href="https://dhruvkb.github.io/pls"
+		thumbnail="dhruvkb.github.io-pls.png"
+	/>
+	<Card
+		title="capo.js"
+		href="https://rviscomi.github.io/capo.js/"
+		thumbnail="capo.js.png"
+	/>
+	<Card
+		title="Web Monetization API"
+		href="https://webmonetization.org/"
+		thumbnail="webmonetization.org.png"
+	/>
+	<Card
+		title="QBCore Docs"
+		href="https://brycerussell.github.io/qbcore-docs/"
+		thumbnail="brycerussell.github.io-qbcore-docs.png"
+	/>
+	<Card title="har.fyi" href="https://har.fyi/" thumbnail="har.fyi.png" />
+	<Card
+		title="xs-dev docs"
+		href="https://xs-dev.js.org"
+		thumbnail="xs-dev.js.org.png"
+	/>
+	<Card
+		title="Felicity"
+		href="https://felicity.pages.dev/"
+		thumbnail="felicity.pages.dev.png"
+	/>
+	<Card
+		title="NgxEditor"
+		href="https://sibiraj-s.github.io/ngx-editor/"
+		thumbnail="sibiraj-s.github.io-ngx-editor.png"
+	/>
+	<Card
+		title="Astro Error Pages"
+		href="https://astro-error-page-documentation.vercel.app/"
+		thumbnail="astro-error-page-documentation.vercel.app.png"
+	/>
+	<Card
+		title="Terrateam Docs"
+		href="https://terrateam.io/docs"
+		thumbnail="terrateam.io-docs.png"
+	/>
+	<Card
+		title="simple-fm"
+		href="https://simple.arciniega.one"
+		thumbnail="simple.arciniega.one.png"
+	/>
+	<Card
+		title="CommandKit"
+		href="https://commandkit.underctrl.io"
+		thumbnail="commandkit.underctrl.io.png"
+	/>
+</FluidGrid>
+
+Voir tous les [dépôts publics de projets utilisant Starlight sur GitHub](https://github.com/withastro/starlight/network/dependents).
+
+## Modules d'extension communautaires
```

Comment:
An official reference for the translation of "plugins" https://www.culture.fr/franceterme/terme/INFO58